packages/api/src/controllers/user.js

```diff
@@ -83,6 +87,10 @@ app.post('/', validatePost('user'), async (req, res) => {
       admin: false,
       emailValidToken: emailValidToken,
       emailValid: validUser,
+      ...(firstName && { firstName }),
```

Comment:
This weird syntax will conditionally add the key value pair to the object. I used this syntax because I wasn't sure if the api would return `firstName: undefined`. If that's the the case and the api excludes undefined values then I can remove the conditional.

Reply:
Confirmed we get rid of this weird syntax. Removed in [04ff25](https://github.com/livepeer/livepeer-com/pull/25/commits/04ff2537a397629316e3ccef0ffba6a97665ad0a)